content/en/docs/sdk/gosdk.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Introduction"
+description: "Introduces the Helm Go SDK"
+aliases: ["/docs/gosdk"]
+weight: 1
+---
+Helm's Go SDK enables custom software to leverage Helm charts and Helm's functionality for managing Kubernetes software deployment (In fact, the Helm CLI is effectively just one such tool!)
+
+Currently, the SDK has been functionally separated from the Helm CLI, and the SDK can (and is) used by standalone tooling. The Helm project has committed to API stability for the SDK. As a warning, the SDK has some rough edges remaining from the initial work to separate the CLI and the SDK. Which the Helm project aims to improve and over time.
+
```

Comment:
may be good to link to the examples page from here. Especially for mobile users—or any user who landed here from a web search—who may not see the examples page in the left navigation.

```suggestion

See [Go SDK Examples]({{< ref "../examples" >}}) for fully working code examples.

```